### PR TITLE
New package: vorbisgain-0.37

### PR DESCRIPTION
**Pull request by Grauwolf**

### CODE REVIEW

File: srcpkgs/vorbisgain/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'vorbisgain'
+pkgname="vorbisgain"
+version="0.37"
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config"
+makedepends="libvorbis-devel"
+depends="libvorbis"
```
Comment (xtraeme) [inline srcpkgs/vorbisgain/template]: This depends line is unnecessary.

File: srcpkgs/vorbisgain/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+short_desc="A utility that computes the ReplayGain values for Ogg Vorbis files"
```
Comment (xtraeme) [inline srcpkgs/vorbisgain/template]: A -> An ?

Comment (Grauwolf) [inline srcpkgs/vorbisgain/template]: the first syllable of the word is pronounced 'you' so 'A utility' should be right

File: srcpkgs/vorbisgain/template (hunk 1 above)
Comment (Grauwolf) [inline srcpkgs/vorbisgain/template]: Sorry. My bad.

**Pull request merged**